code/datums/components/religious_tool.dm

```diff
@@ -124,11 +125,21 @@
 		message_admins("[ADMIN_LOOKUPFLW(usr)] has tried to spawn an item when selecting a sect.")
 		return
 	if(user.mind.holy_role != HOLY_ROLE_HIGHPRIEST)
-		to_chat(user, "<span class='warning'>You are not the high priest, and therefore cannot select a religious sect.")
+		to_chat(user, "<span class='warning'>You are not the high priest, and therefore cannot select a religious sect.</span>")
 		return
 	if(!user.canUseTopic(parent, BE_CLOSE, FALSE, NO_TK))
 		to_chat(user, "<span class='warning'>You cannot select a sect at this time.</span>")
 		return
+	var/mob/living/carbon/human/testsubject
+	var/datum/religion_sect/pathed = path
+	if(user == testsubject)
+		if(testsubject.dna.species.id == pathed.req_species)
+			pathed.starter = TRUE
+			pathed.sect_requirements = "Unlocked"//////////////////////////////// work on this path shit ugh
+			to_chat(user,"<span class='big notice'>Congratulations, you've unlocked [pathed.name]!</span>")
+	if(!pathed.starter)
+		to_chat(user, "<span class='warning'>You cannot select this sect until you complete its requirements.</span>")
+		return
```

Comment:
no, no, you can handle that in `generate_available_sects()` (but I suggested something wrong before.) Revoke this part fully.